config/routes.rb

```diff
@@ -1,10 +1,11 @@
 Rails.application.routes.draw do
-  get 'users/new'
   scope "(:locale)", locale: /en|vi/ do
     root "static_pages#home"
-    get  "/help",    to: "static_pages#help"
-    get  "/about",   to: "static_pages#about"
+    get  "/help", to: "static_pages#help"
```

Comment:
```suggestion
    get "/help", to: "static_pages#help"
```
giữa các phần chỉ để lại 1 space